tests/Doctrine/Migrations/Tests/DependencyFactoryTest.php

```diff
@@ -156,6 +158,36 @@ public function testServiceDefinition() : void
         self::assertSame($anotherLogger, $di->getLogger());
     }
 
+    public function testServiceDefinitionWithReference() : void
+    {
+        $migration = $this->createMock(AbstractMigration::class);
+
+        $di = DependencyFactory::fromConnection(new ExistingConfiguration($this->configuration), new ExistingConnection($this->connection));
```

Comment:
What does `di` mean here? Dependency injection? Shouldn't it be `$dependencyFactory`?